modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc

```diff
@@ -14,6 +14,13 @@ Available tests to run:
 ** Throughput test: 8192-bit messages
 *** Unique pairs of Redpanda nodes each act as a client and a server.
 *** The test pushes as much data over the wire, within the test parameters.
+* *Cloud tests*
+** Latency test: 1024-bit object.
+** Depending on cluster read/write permissions (xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_read[`cloud_storage_enable_remote_read`], xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_write[`cloud_storage_enable_remote_write`]), a series of cloud storage operations are performed:
+*** Upload an object to an S3 bucket.
+*** List objects in the bucket.
+*** Download an object from the bucket.
```

Comment:
```suggestion
*** Download an object from the S3 bucket.
```

Reply:
Is this feature S3 specific ? @r-vasquez

Reply:
I confirmed with Core that this feature accepts either ABS or S3, so we need to remove any reference to `S3` in this new paragraph.

I'll make sure to bring this back to the rpk code.

Reply:
Changed from s3 bucket to object storage